freud/plot.py

```diff
@@ -142,20 +196,76 @@ def pmft_plot(pmft, ax=None):
         ax.yaxis.set_ticks([i for i in range(int(ylims[0]), int(ylims[1]+1))])
         ax.set_xlabel(r'$x$')
         ax.set_ylabel(r'$y$')
-
         ax.set_title('PMFT')
+
+        ax_divider = make_axes_locatable(ax)
+        cax = ax_divider.append_axes("right", size="7%", pad="10%")
+
         im = ax.imshow(np.flipud(pmft_arr),
                        extent=[xlims[0], xlims[1], ylims[0], ylims[1]],
                        interpolation='nearest', cmap='viridis',
                        vmin=-2.5, vmax=3.0)
 
-        ax_divider = make_axes_locatable(ax)
-        cax = ax_divider.append_axes("right", size="7%", pad="10%")
-        cb = colorbar(im, cax=cax)
-        cb.set_label_text(r"$k_B T$")
+        cb = Colorbar(cax, im)
+        cb.set_label(r"$k_B T$")
+
         return ax
 
 
+def plot_density(density, box, ax=None):
+    R"""Helper function to plot density diagram.
+
+    Args:
+        density (:math:`\left(N_x, N_y\right)` :class:`numpy.ndarray`):
+            Array containing density.
+        box (:class:`freud.box.Box`):
+            Simulation box.
+        ax (:class: matplotlib.axes.Axes): axes object to plot on.
+            If :code:`None`, make a new axes and figure object.
+            (Default value = :code:`None`).
+
+    Returns:
+        :class: matplotlib.axes.Axes: axes object with the diagram.
+    """
+    if not MATPLOTLIB:
+        return None
+    try:
+        from mpl_toolkits.axes_grid1.axes_divider import make_axes_locatable
+        from matplotlib.colorbar import Colorbar
+        # import matplotlib.transforms
```

Comment:
Let's remove this and the other commented code - I have changed my mind about #319 for now. I'm no longer certain that we'll need this feature. We can always pull it back from the git history if we need it someday.